DFC.ServiceTaxonomy.GraphSync/GraphSyncers/GraphSyncer.cs

```diff
@@ -83,6 +90,24 @@ public async Task SyncToGraph(ContentItem contentItem)
             await SyncComponentsToGraph(graphSyncPartContent);
         }
 
+        public async Task DeleteFromGraph(ContentItem contentItem)
+        {
+            _deleteNodeCommand.ContentType = contentItem.ContentType;
+            _deleteNodeCommand.Uri = contentItem.Content.GraphSyncPart.Text;
```

Comment:
We should silently handle the case where the content item being deleted hasn't got a graph sync part, i.e. do nothing.
We don't whitelist content types in the triggering event and use the presence of a graph sync part as a flag to sync the content item.